planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module.hpp

```diff
@@ -68,13 +68,13 @@ class AvoidanceModule : public SceneModuleInterface
   }
 
 private:
-  struct RegisteredShiftPoint
+  struct RegisteredShiftLine
```

Comment:
To me, segment sounds better than line. What do you think.

Reply:
Initially, I think segment is better, but more people agree with `line`, so we decided to use ShiftLine

Reply:
Ok, got it.